config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits.yaml

```diff
@@ -36,7 +36,7 @@ presubmits:
         command:
         - scripts/test-infra/verify.sh
   - name: pull-node-feature-discovery-verify-docs
-    always_run: true
+    skip_if_only_changed: "(((^|/)OWNERS))$"
```

Comment:
I think we could invert this so that it's run ONLY IF `docs/` has changed.🤔 We run mdlint over stuff under docs/